graphs/energy/nodes/energy/energy_chp_supercritical_ccs_waste_mix.ad

```diff
@@ -0,0 +1,38 @@
+- output.loss = elastic
+- groups = [preset_demand, central_production, cost_chps, electricity_production, dispatchable_production, heat_production, hv_net_supply, wacc_proven_tech, primary_heat_infrastructure]
+- use = energetic
+- presentation_group = chps
+- graph_methods = [demand, full_load_hours]
+- waste_outputs = [steam_hot_water]
+- availability = 0.9
+- ccs_investment = 69000000.0
+- construction_time = 3.5
+- cost_of_installing = 0.0
+- decommissioning_costs = 0.0
+- fixed_operation_and_maintenance_costs_per_year = 4800000.0
+- initial_investment = 162000000.0
+- technical_lifetime = 30.0
+- variable_operation_and_maintenance_costs_for_ccs_per_full_load_hour = 0.0
+- variable_operation_and_maintenance_costs_per_full_load_hour = 0.0
+- wacc = 0.04
+- energy_balance_group = central power generation
+- heat_network.group = self: electricity_output_curve
+- heat_network.subtype = must_run
+- heat_network.type = producer
+- merit_order.level = hv
+- merit_order.subtype = dispatchable
+- merit_order.type = producer
+- electricity_output_capacity = 60.0
```

Comment:
@AlexanderWirtz and I found an issue with CHPs with dynamic efficiencies (see https://github.com/quintel/etengine/issues/1189). To avoid this, could you remove `electricity_output_capacity` and `heat_output_capacity` from this file and replace it with `typical_input_capacity`? `typical_input_capacity` should be equal to `electricity_output_capacity` divided by a typical/sensible value for `output.electricity`. I guess this also needs to be changed in the node source analysis on ETDataset.